content/swift/concepts/methods/methods.md

```diff
@@ -0,0 +1,155 @@
+#Methods
```

Comment:
Missing metadata! Please compare with the template:

https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md

Reply:
No worries. Will make the changes later or tomorrow and update the PR 🚧

Reply:
How's it looking? Anything else needing changes? @yangc95